src/helper_addressbook.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```

Comment:
Why shebang here?

Reply:
Another helper ):

Reply:
Ok.

Reply:
I mean, I agree about the shebang, but the addressbook code shouldn't be in the UI like it is now. What other options are there? Not another thread to deal with addressbook?

Reply:
I agree that it's cool to have all CRUD tasks (on messages, addressbook and black/white lists) in the main package to use them in `api`, `bitmessageqt` and any other frontend. Also I was working on the single helper with all SQL queries in #1534 because it is bad to have SQL scattered amond the code. It needs heavy testing. Currently it breaks `bitmessageqt` because I missed some type cast or check.

But hey, you need this helper now only because of `BITMESSAGE_HOME` in env and the absence of cleanup.

Another thread is the last thing I would suggest. But recently I thought about using multiprocessing for test cases which use potentially dangerous imports.